lib/commanded/process_manager.ex

```diff
@@ -38,17 +38,25 @@ defmodule OpentelemetryCommanded.ProcessManager do
     :otel_propagator_text_map.extract(trace_headers)
 
     attributes = [
-      application: meta.application,
-      "process_manager.uuid": meta.process_uuid,
-      "process_manager.name": meta.process_manager_name,
-      "process_manager.module": meta.process_manager_module,
-      "event.id": event.event_id,
-      "event.number": event.event_number,
-      "event.type": event.event_type,
-      "correlation.id": event.correlation_id,
-      "causation.id": event.causation_id,
-      "stream.id": event.stream_id,
-      "stream.version": event.stream_version
+      "messaging.system": "commanded",
+      "messaging.protocol": "cqrs",
+      "messaging.destination_kind": "process_manager",
+      "messaging.operation": "receive",
```

Comment:
@bryannaegele 
A Process Manager "receives" events and "sends" commands.
Is this correct?

Reply:
@bryannaegele 
Sort of.  It `receive`s events, and its `handle` call may return Commands.
The Process Manager does not `send` any Commands it produces instead Commanded hands them back to the app to be dispatched